obs-tools/opensuse42_submit.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+LIST=`osc list openSUSE:Factory | grep -i -e yast2 -e skelcd -e "^libstorage$" -e perl-bootloader -e linuxrc`
+
+for i in $LIST; do
+  # compare if something is new by comparing binaries ( as version is part of rpm output )
+  OS42=$(osc list -b openSUSE:42 $i -r standard -a x86_64)
+  FACTORY=$(osc list -b openSUSE:Factory $i -r standard -a x86_64)
+  if [ $OS42 == $FACTORY ]; then
```

Comment:
Um, I'm not sure if this exact comparing is OK. For example `osc list -b openSUSE:Factory yast2 -r standard -a x86_64` returns

```
_buildenv
_statistics
rpmlint.log
yast2-3.1.142-1.1.src.rpm
yast2-3.1.142-1.1.x86_64.rpm
yast2-devel-doc-3.1.142-1.1.x86_64.rpm
yast2.desktopfiles
```

The RPM name includes also a build number so it might create some unnecessary submit requests if a package is rebuilt in Factory but not in 42 (or the other way round).